<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>673</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>档案人脸识别系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>目录</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="dirpushButton">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+O</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QRadioButton" name="all_pic_radioButton">
      <property name="text">
       <string>显示所有图片</string>
      </property>
      <property name="shortcut">
       <string>Alt+Q</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QRadioButton" name="part_recognition_radioButton">
      <property name="text">
       <string>显示部分识别图片</string>
      </property>
      <property name="shortcut">
       <string>Alt+W</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QRadioButton" name="all_recognition_radioButton">
      <property name="text">
       <string>显示全部识别图片</string>
      </property>
      <property name="shortcut">
       <string>Alt+E</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>4</y>
         <width>40</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>档号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="arch_num_lineEdit">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>4</y>
         <width>201</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>主题：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="3">
     <widget class="QTextEdit" name="theme_textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">   QHeaderView::section{
            border-top:0px solid #D8D8D8;
            border-left:0px solid #D8D8D8;
            border-right:1px solid #D8D8D8;
            border-bottom: 1px solid #D8D8D8;
            background-color:white;
            padding:4px;
        }
        QTableCornerButton::section{
            border-top:0px solid #D8D8D8;
            border-left:0px solid #D8D8D8;
            border-right:1px solid #D8D8D8;
            border-bottom: 1px solid #D8D8D8;
            background-color:white;
        }</string>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>89</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>编辑</string>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QCheckBox" name="verifycheckBox">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>已核验</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+S</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="preButton">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;- 上一张</string>
      </property>
      <property name="shortcut">
       <string>Left</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QLabel" name="pic_index_label">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="nextButton">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>下一张 -&gt;</string>
      </property>
      <property name="shortcut">
       <string>Right</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="9" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="recognitionTab">
       <attribute name="title">
        <string>    识别    </string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="run_state_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="recognized_face_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="recognition_rate_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="handled_pic_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>全部识别图片：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="all_recognized_pic_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>状态：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>整体识别率：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>已识别人物：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>部分识别图片：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>未处理图片：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="unhandled_pic_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string>已处理图片：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="part_recognized_pic_label">
          <property name="font">
           <font>
            <family>新宋体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="recogniButton">
          <property name="text">
           <string>识别</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="pausecontinueButton">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pictureTab">
       <attribute name="title">
        <string>    图片    </string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="pic_view">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="archcodeTab">
       <attribute name="title">
        <string>预设档号</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QPushButton" name="add_folder_btn">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="cancel_folder_btn">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">文件夹</string>
           </property>
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>档号</string>
           </property>
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modeltrainTab">
       <attribute name="title">
        <string>模型训练</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="2">
         <widget class="QPushButton" name="train_pushButton">
          <property name="text">
           <string>开始训练</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>阈值：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="thresh_lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>欧氏距离：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="distance_lineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="9" column="3" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>新宋体</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(141, 141, 141);</string>
      </property>
      <property name="text">
       <string>提示：Ctrl+S保存已核验</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
